src/main/java/io/snyk/jenkins/SnykStepBuilder.java

```diff
@@ -265,7 +264,7 @@ public FormValidation doCheckSnykTokenId(@AncestorInPath Item item, @QueryParame
         return FormValidation.warningWithMarkup("A Snyk API token is required. If you do not provide credentials, make sure to provide a <code>SNYK_TOKEN</code> build environment variable.");
       }
 
-      if (null == CredentialsMatchers.firstOrNull(lookupCredentials(SnykApiToken.class, Jenkins.get(), ACL.SYSTEM, Collections.emptyList()),
+      if (null == CredentialsMatchers.firstOrNull(lookupCredentialsInItemGroup(SnykApiToken.class, Jenkins.get(), ACL.SYSTEM2, Collections.emptyList()),
```

Comment:
Jenkins 2.479.x introduces Spring Security 6. The former method and field was deprecated and replaced by `SYSTEM2` and `lookupCredentialsInItemGroup` respectively.